### PR TITLE
Added new query functions

### PR DESCRIPTION
**Pull request by jadchaar**
I have added five new query functions to complement the recent addition of `is_between`. I feel that these additions will better align Arrow with Moment's query function interface.

One thing I would like feedback on: I did not know what to call the function argument other than "target". Any ideas for a better variable name is appreciated. Thanks!

### CODE REVIEW

File: test.py [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import arrow
```
Comment (jaapz) [inline test.py]: Looks like you forgot to remove a test file here!

**Comment by jadchaar** [comment]
@systemcatch thanks for the feedback. Are you referring to the overrided operators in `arrow.py`?

If so, I thought these would be a nice abstraction over those operators and a more friendly interface for those that are coming from Moment.js.

If you still think this does not add much value, feel free to close this PR and let me know what I could work on next :D.

**Comment by systemcatch** [comment]
Yes I was referring to those operators, though I now understand what you wanted to achieve with these changes.

Personally I think python's syntax for this kind of thing is already quite user friendly, the is_between() is useful though. Let's leave this out for now.

I will have a think on some good issues to tackle.

**Comment by jadchaar** [comment]
Would it be possible to triage some issues that you think should be tackled for the next release? I'd love to help out!

**Comment by systemcatch** [comment]
> Would it be possible to triage some issues that you think should be tackled for the next release? I'd love to help out!

Sent you an email!

**Pull request closed without merging**